docs/design/proposals/carm/cross-account-resource-management.md

```diff
@@ -65,17 +73,43 @@ metadata:
   name: ack-carm-config-map
   namespace: default
 data:
-  accounts: |
-    123456789012=arn:aws:iam::123456789012:root
-    454545454545=arn:aws:iam::454545454545:role/S3Access
+  accounts:
+    "123456789012": arn:aws:iam::123456789012:root
+    "454545454545": arn:aws:iam::454545454545:role/S3Access
 ```
 
+In controller-runtime, the reconciler will need to lookup the `ack-carm-config-map` data content to query the value of a particular key. To realise that, the controllers will need keep a cached version of the ConfigMap and frequently update it when changes are made by cluster admins.
```

Comment:
Let's say "ACK runtime" here, instead of "controller-runtime", since the upstream Kubernetes `controller-runtime` is a package itself.